src/Interpreters/BlockUtils.cpp

```diff
@@ -103,13 +103,23 @@ Block buildBlock(
     return block;
 }
 
-void appendBlock(Block && block, ContextPtr context, IDistributedWriteAheadLog::OpCode opCode, const Poco::Logger * log)
+void appendDDLBlock(
+    Block && block,
+    ContextPtr context,
+    const std::vector<String> & parameter_names,
+    IDistributedWriteAheadLog::OpCode opCode,
+    const Poco::Logger * log)
 {
     IDistributedWriteAheadLog::Record record{opCode, std::move(block)};
     record.headers["_version"] = "1";
-    if (context->getQueryParameters().contains("table_type"))
+
```

Comment:
Nit : `const auto & query_params = context->getQueryParameters();` and reuse `query_params` in the loop

Reply:
fixed